docs_generator/reference_docs_generator.py

```diff
@@ -11,7 +11,7 @@ def save_to_markdown_file(filepath: Path, content: str):
         with open(filepath, "w", encoding="utf_8") as file:
             file.write(content)
 
-    def __init__(self, cli_app: CLIApp) -> None:
+    def __init__(self, cli_app: CLIApp[Any]) -> None:
```

Comment:
This highlights the issue with `CLIApp` being generic IMHO.

Reply:
Fixed